manager/manifests/istio.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+gateways:
+  enabled: true
+  istio-ingressgateway:
+    enabled: false
+  operator-ingressgateway:
+    namespace: istio-system
+    enabled: true
+    labels:
+      app: operator-istio-gateway
+      istio: operator-ingressgateway
+    type: LoadBalancer
+    ports:
+    - port: 80
+      targetPort: 80
+      name: http2
+    - port: 443
+      name: https
+    - port: 31400
+      name: tcp
+    - port: 15011
+      targetPort: 15011
+      name: tcp-pilot-grpc-tls
+    - port: 8060
+      targetPort: 8060
+      name: tcp-citadel-grpc-tls
+    secretVolumes:
+    - name: customgateway-certs
+      secretName: istio-customgateway-certs
+      mountPath: /etc/istio/customgateway-certs
+    - name: customgateway-ca-certs
+      secretName: istio-customgateway-ca-certs
+      mountPath: /etc/istio/customgateway-ca-certs
+  apis-ingressgateway:
+    namespace: istio-system
+    enabled: true
+    labels:
+      app: apis-istio-gateway
+      istio: apis-ingressgateway
+    type: LoadBalancer
+    ports:
+    - port: 80
+      targetPort: 80
+      name: http2
+    - port: 443
+      name: https
+    - port: 31400
+      name: tcp
+    - port: 15011
+      targetPort: 15011
+      name: tcp-pilot-grpc-tls
+    - port: 8060
+      targetPort: 8060
+      name: tcp-citadel-grpc-tls
+    secretVolumes:
+    - name: customgateway-certs
+      secretName: istio-customgateway-certs
+      mountPath: /etc/istio/customgateway-certs
+    - name: customgateway-ca-certs
+      secretName: istio-customgateway-ca-certs
+      mountPath: /etc/istio/customgateway-ca-certs
+
+sidecarInjectorWebhook:
+  enabled: true
+  image: $CORTEX_IMAGE_ISTIO_SIDECAR
+  alwaysInjectSelector:
+    - matchLabels:
+        workloadType: api
+    - matchLabels:
+        workloadType: operator
+
+galley:
+  image: $CORTEX_IMAGE_ISTIO_GALLEY
```

Comment:
Did you confirm this pulls from the overridden value? According to https://istio.io/docs/reference/config/installation-options/, the default value of this config is "galley", which does not include the full URL (presumably you would also set "hub". But maybe if you pass in the full URL it figures it out?

Reply:
it does, in the actual template it checks for "/" https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml#L38